client/src/components/new-request/main-container.vue

```diff
@@ -65,7 +65,7 @@ export default class MainContainer extends Mixins(SessionTimerMixin, DisplayedCo
   }
 
   backToSearch () {
-    if ([2, 3].includes(newReqModule.submissionTabNumber) &&
+    if ([2, 3].includes(newReqModule.submissionTabNumber) && !this.editMode &&
```

Comment:
Did you verify these pages in edit mode?

Reply:
yes

Reply:
Excellent!